src/rules/order.js

```diff
@@ -384,6 +435,15 @@ module.exports = {
               'never',
             ],
           },
+          alphabetize: {
+            type: 'object',
```

Comment:
```suggestion
            type: 'object',
            additionalProperties: false,
```

Reply:
What is the change request here? To move the `additionalProperties: false` directly under `type: 'object'`, instead of its current position at the bottom of the schema (under `properties`)?